index.js

```diff
@@ -498,10 +498,18 @@ prototype.buildOperation = function(oldOperation, operationDefaults) {
     parameters.push(this.buildParameter(oldParameter));
   });
 
+  var tags = (operationDefaults.tags || []).concat(oldOperation.tags || []);
+
+  tags = tags.filter(function(e, i, arr) {
```

Comment:
From 503 to 506 is to filter duplicates, is like a "uniq!"

http://www.w3schools.com/jsref/jsref_lastindexof.asp